libs/contrib/nanovg_helper.xtm

```diff
@@ -0,0 +1,742 @@
+;;; nanovg_helper.xtm -- nanovg helper library
+
+;; Author: Yuya Chiba (cy.blue.9@gmail.com)
+;; Keywords: extempore
+;; Required dylibs: libnanovg
+
+;;; Commentary:
+
+;; This library is warrper library of nanovg(libs/external/nanovg.xtm)
```

Comment:
might want to fix the typo in wrapper while you're at it 😉

Reply:
Sorry for making a easy mistake. 😓